include/veriblock/pop/mock_miner.hpp

```diff
@@ -119,7 +119,12 @@ class MockMiner {
   const VbkChainParams& vbkParams() const { return vbk_params_; }
   const BtcChainParams& btcParams() const { return btc_params_; }
 
-  MockMiner() {
```

Comment:
You could keep the default constructor and avoid some duplicate code.

Reply:
I am not sure that is a good idea, the main purpose is it externally setup MockMiner, so if you want to init it with RegTest version you should do it explicitly